flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -431,9 +432,9 @@ struct IntrinsicLibrary {
   mlir::Value genIbits(mlir::Type, llvm::ArrayRef<mlir::Value>);
   mlir::Value genIbset(mlir::Type, llvm::ArrayRef<mlir::Value>);
   fir::ExtendedValue genIchar(mlir::Type, llvm::ArrayRef<fir::ExtendedValue>);
-  mlir::Value genIEOr(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genIeor(mlir::Type, llvm::ArrayRef<mlir::Value>);
   fir::ExtendedValue genIndex(mlir::Type, llvm::ArrayRef<fir::ExtendedValue>);
-  mlir::Value genIOr(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genIor(mlir::Type, llvm::ArrayRef<mlir::Value>);
```

Comment:
These renaming games beg the question again. Why `CpuTime` instead of `Cputime`?

Reply:
It's a minor choice to fit within the now slightly fewer arbitrary variations.